packages/valory/connections/abci/tests/test_abci.py

```diff
@@ -79,6 +85,19 @@
 from packages.valory.protocols.abci.dialogues import AbciDialogues as BaseAbciDialogues
 
 
+PACKAGE_DIR = Path(__file__).parent.parent
+
+
+@pytest.fixture(scope="session", autouse=True)
+def hypothesis_cleanup() -> Generator:
+    """Fixture to remove hypothesis directory after tests."""
+    yield
+    hypothesis_dir = PACKAGE_DIR / ".hypothesis"
+    if hypothesis_dir.exists():
+        with suppress(OSError, PermissionError):
+            shutil.rmtree(hypothesis_dir)
```

Comment:
`.hypothesis` directory gets created in the package directory every time we run tests which leads to hash related issue so this fixture will perform a clean up after the test session is over.